README.md

```diff
@@ -79,7 +79,7 @@ All contributors to any Work Items must be members of the Solid CG. It’s easy
 
 Anyone can join the [Solid WebID Profile chat](https://gitter.im/solid/webid-profile).
 
-Mondays, 15:00 UTC at https://meet.jit.si/solid-profile . Meetings are transcribed and [published](https://github.com/solid/webid-profile/tree/main/meetings/).
+A **tentative** meeting slot is held open on Mondays, 15:00 UTC at https://meet.jit.si/solid-profile. Please refer to the [chat](https://gitter.im/solid/webid-profile) for concrete meeting plans and agendas. Meetings are transcribed and [published](https://github.com/solid/webid-profile/tree/main/meetings/).
```

Comment:
We should suggest people can join the Solid CG meetings if they'd like to discuss a WebID-Profile topic.

Reply:
Do you mean as a definitive decision to not have WebID Profile specific meetings anymore? Or as an additional suggestion to this note?

Reply:
Additionally. As in another avenue to bring up a WebID-Profile topic for discussion, aside from the chat.